src/psycop_feature_generation/text_models/text_model_pipeline.py

```diff
@@ -84,23 +82,24 @@ def text_model_pipeline(
             filename,
         )
 
-    # create query str
-    query = f"SELECT * FROM fct.{view}"
-
-    if sfi_type != ("All_sfis",):
-        query += f" WHERE overskrift in ('{sfi_type[0]}'"
-        if len(sfi_type) > 1:
-            for sfi in sfi_type[1:]:
-                query += f",'{sfi}'"
-        query += ")"
-
     # load corpus
-    corpus = sql_load(query=query, n_rows=n_rows)
+    # if not sfi_type, include all sfis
+    if not sfi_type:
+        corpus = pd.read_parquet(
+            path=f"E:/shared_resources/preprocessed_text/{corpus_name}.parquet",
```

Comment:
```suggestion
            path=f"E:/shared_resources/preprocessed_text/{corpus_name}.parquet", # TODO: Use SHARED_RESOURCES_PATH
```

In psycop_common, we have a global variable for shared resources. Just a TODO for me to remember to do that when merging :-)